signer/src/transaction_signer.rs

```diff
@@ -534,63 +548,90 @@ where
                 if let Some(pause) = self.dkg_begin_pause {
                     // Let's give the others some slack
                     tracing::debug!(
-                        "Sleeping a bit to give the other peers some slack to get DkgBegin"
+                        "sleeping a bit to give the other peers some slack to get dkg-begin"
```

Comment:
Why change to `dkg-begin`?

Reply:
just to stay consistent with keeping things lowercase in logs, `dkgbegin` or `signaturesharerequest` are harder to read

Reply:
I like keeping struct names. I wouldn't have expected to change to lowercase with dashes when grepping.

Reply:
My vote is on `dkg-begin`, since it's easier to type.

Reply:
We'll go with kebab-case for now. If it's a problem then we can revert it.